### PR TITLE
Initial StinkBomb commit!

### PR DESCRIPTION
**Pull request by golmansax**

### CODE REVIEW

**Comment by golmansax** [comment]
@mockdeep you will need to subscribe to this repo

**Comment by golmansax** [comment]
@mockdeep @QuotableWater7 bump on this pull

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.PHONY: test coverage codeclimate
```
Comment (mockdeep) [inline Makefile:1]: A Makefile, eh?

**Comment by QuotableWater7** [comment]
sure lgtm

File: Makefile (hunk 1 above)
Comment (golmansax) [inline Makefile:1]: Haha yeah.  Either that or grunt (which makes me gulp...)

**Pull request merged**